bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md

```diff
@@ -0,0 +1,107 @@
+# Paper review-Perception Aware Model Predictive Control for Quadrotors 
+
+Authors - Davide Falanga, Philipp Foehn, Peng Lu, and Davide Scaramuzza
+
+## Why is this important?
+
+As we are advancing into a world of autonomous control of vehicles, there have been some key developments that have accelerated the research in the domain. One of these, is the Perception Aware Model Predictive Control (PAMPC) for Quadrotors, which I am going to review in this document. The focus of this review is to summarize the Model Predictive Control (MPC) pipeline used along with its cost function and control constraints.
+
+## Introduction
+
+Previously, perception and control action were treated as two separate problems in controls. To solve this problem, the authors of the paper designed a Model Predictive Control that optimizes the control input for trajectory optimization and simultaneously optimizing the perception of the sensing algorithms to maximize the vision needed to execute a control action. An example of this can be the path planning and trajectory optimization for the quadrotor to pass through a narrow gap, at the same time making sure the gap is always visible. A big advantage of this approach is that it is independent of the task and can be used in variety of other applications, other than quadrotors, that involve a vision-based localization and target tracking.
+
+## Methodology
```

Comment:
This part has a good flow and labelling in all the equation. It shows detailed mathematical representation of how perception is integrated into the MPC framework. It would be really good if the writer can give step-by-step explanations or a flowchart describing how each component interacts within the system.